src/lavinmq/auth/user.cr

```diff
@@ -14,130 +11,17 @@ module LavinMQ
       forward_missing_to @cache
     end
 
-    class User
-      include SortableJSON
-      getter name, password, permissions
-      property tags, plain_text_password
+    abstract class User
       alias Permissions = NamedTuple(config: Regex, read: Regex, write: Regex)
+      alias CacheKey = Tuple(String, String)
```

Comment:
Already definied on line 5, but maybe remove that definition as CacheKey is only used in this class currenlty.